projects/dotcms-field-elements/src/components/dot-date/dot-date.tsx

```diff
@@ -88,6 +95,15 @@ export class DotDateComponent {
         this.validateProps();
     }
 
+    componentDidLoad(): void {
+        const htmlElement = this.el.querySelector('input');
+        setTimeout(() => {
+            const attrs: Attr[] = Array.from(this.el.attributes);
+            const attrException = new Array('dottype'.toUpperCase());
+            setAttributesToElement(htmlElement, attrs, attrException);
```

Comment:
Why passing `attrException`? can't just `filter` and pass exactly what attrs that can be set?